src/Chapter1.hs

```diff
@@ -632,8 +646,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = lastDigit (div (abs n) 10) + lastDigit n
+  where
+    lastDigit :: Int -> Int
+    lastDigit x = mod (abs x) 10
```

Comment:
Nice catch!

You can notice, that actually this function is implemented as one of the previous tasks, so you can actually reuse it here 👍🏼 

Alternatively, we can notice that both of these combinations are used in this function implementation:

```haskell
mod (abs n) 10
div (abs n) 10
```

One hint to make the solution shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!

So you could write it this way:

```haskell
(x, y) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂 
And you can use `x` and `y` later instead 👍🏼

Reply:
Yes, I remembered it from the task before, but this task said to do solve it with let-in or where and this seemed like the most straight-forward way to me 😅 

The divMod took me a hot minute to understand, but after some playing around with it a bit in GHCi I think I understand it and I added a second solution to this task using divMod 😄